libs/agno/agno/tools/openai.py

```diff
@@ -36,3 +46,69 @@ def transcribe_audio(self, audio_path: str) -> str:
         except Exception as e:
             logger.error(f"Failed to transcribe audio: {str(e)}")
             return f"Failed to transcribe audio: {str(e)}"
+
+    def generate_image(self, prompt: str, n: int = 1, size: OpenAIImageSize = "1024x1024") -> str:
```

Comment:
Lets remove "n" as a param and just hard code it, so the model can't get it wrong?